book/website/project-design/data-governance/data-gov-ml.md

```diff
@@ -0,0 +1,67 @@
+(pd-dg-ml)=
+
+# Data Governance for the Machine Learning Pipeline
+
+The Machine Learning (ML) pipeline consists of a series of activities including the collection of data, training of an ML model, and the deployment of the model into use. 
+Data is integral throughout the ML process and the methods for which data is collected, annotated, processed, and shared will impact individuals who may be the subjects in or creators of the data as well as communities represented in the data.
```

Comment:
I'm wondering if perhaps this should be separated out into the subjects (for whom privacy is a concern but not necessarily access) and people who want access to the completed datasets?